src/main/java/controller/LadderGameController.java

```diff
@@ -0,0 +1,53 @@
+package controller;
+
+import domain.Height;
+import domain.Ladder;
+import domain.Name;
+import domain.Names;
+import dto.GameDto;
+import util.RandomBooleanGenerator;
+import view.InputView;
+import view.OutputView;
+
+public class LadderGameController {
+
+    public void run() {
+        Names names = getNames();
+        Height height = getHeight();
+
+        Ladder ladder = buildLadder(names, height);
+
+        printResult(names, ladder);
+    }
+
+    private Ladder buildLadder(final Names names, final Height height) {
+        Ladder ladder = new Ladder(new RandomBooleanGenerator());
+        ladder.build(height, names.count());
+        return ladder;
+    }
+
+    private static void printResult(final Names names, final Ladder ladder) {
+        OutputView.printResult(new GameDto(names, ladder));
+    }
+
+
+    private Names getNames() {
+        try {
+            OutputView.printRequestNames();
+            return new Names(Name.of(InputView.getNames()));
```

Comment:
아래와 같이 하는 경우와는 어떤 차이가 있나요?
```suggestion
            return Names.of(InputView.getNames());
```

Reply:
제안해주신 방식이 더 가독성 좋은 것 같아요! 사실 생성자를 통한 객체 생성과 정적 팩토리 메소드를 통한 객체 생성 방식이 통일되어 있지 않아서 리팩토링 대상이었는데, 짚어주셔서 감사합니다! 이번 리팩토링 작업에서 통일할 생각이에요.